package.json

```diff
@@ -41,63 +42,65 @@
     "lint:security": "nsp check"
   },
   "devDependencies": {
-    "autoprefixer": "7.1.6",
-    "babel-core": "6.26.0",
-    "babel-eslint": "7.2.3",
-    "babel-jest": "20.0.3",
-    "babel-loader": "7.1.2",
+    "autoprefixer": "9.0.0",
+    "babel-core": "6.26.3",
+    "babel-eslint": "8.2.6",
+    "babel-jest": "23.4.0",
+    "babel-loader": "7.1.5",
     "babel-plugin-import": "1.8.0",
     "babel-plugin-transform-decorators-legacy": "1.3.5",
     "babel-polyfill": "6.26.0",
-    "babel-preset-react-app": "3.1.1",
+    "babel-preset-react-app": "3.1.2",
     "babel-runtime": "6.26.0",
-    "case-sensitive-paths-webpack-plugin": "2.1.1",
-    "chalk": "1.1.3",
-    "css-loader": "0.28.7",
-    "dotenv": "4.0.0",
+    "case-sensitive-paths-webpack-plugin": "2.1.2",
+    "chalk": "2.4.1",
+    "css-loader": "1.0.0",
+    "cssnano": "4.0.3",
+    "dotenv": "6.0.0",
     "dotenv-expand": "4.2.0",
     "enzyme": "3.3.0",
     "enzyme-adapter-react-16": "1.1.1",
-    "eslint": "4.10.0",
+    "eslint": "5.2.0",
     "eslint-config-prettier": "2.9.0",
-    "eslint-config-react-app": "2.1.0",
-    "eslint-loader": "1.9.0",
-    "eslint-plugin-flowtype": "2.39.1",
-    "eslint-plugin-import": "2.8.0",
-    "eslint-plugin-jsx-a11y": "5.1.1",
+    "eslint-config-react-app": "3.0.0-next.3e165448",
+    "eslint-loader": "2.1.0",
+    "eslint-plugin-flowtype": "2.50.0",
+    "eslint-plugin-import": "2.13.0",
+    "eslint-plugin-jsx-a11y": "6.1.1",
     "eslint-plugin-prettier": "2.6.2",
-    "eslint-plugin-react": "7.4.0",
-    "extract-text-webpack-plugin": "3.0.2",
+    "eslint-plugin-react": "7.10.0",
+    "extract-css-chunks-webpack-plugin": "3.0.10",
     "faker": "4.1.0",
     "fetch-mock": "6.5.2",
-    "file-loader": "1.1.5",
-    "fs-extra": "3.0.1",
+    "file-loader": "1.1.11",
+    "fs-extra": "7.0.0",
     "generate-json-webpack-plugin": "0.3.1",
-    "html-webpack-plugin": "2.29.0",
-    "jest": "20.0.4",
+    "html-webpack-plugin": "3.2.0",
+    "jest": "23.4.1",
     "jest-immutable-matchers": "2.0.1",
-    "less": "3.7.1",
+    "less": "3.8.0",
     "less-loader": "4.1.0",
     "nsp": "3.2.1",
     "object-assign": "4.1.1",
-    "postcss-flexbugs-fixes": "3.2.0",
-    "postcss-loader": "2.0.8",
+    "postcss-flexbugs-fixes": "4.0.0",
+    "postcss-loader": "2.1.6",
     "prettier": "1.13.7",
```

Comment:
Just FYI prettier is now up to 1.14.0 according to https://github.com/mozilla/delivery-console/pull/324

Reply:
yeah I left that one open so that we could upgrade there

Reply:
i mostly wanted to get webpack and the other packages that were exposed by ejecting upgraded.